src/swap/types.ts

```diff
@@ -345,3 +348,12 @@ export type Transaction = {
   to: Address; // The recipient address
   value: bigint; // The value of the transaction
 };
+
+export type UseAwaitCallsParams = {
+  accountConfig: Config;
+  config: {
+    maxSlippage: number;
+  };
```

Comment:
Here we can use the type `SwapConfig`:

```
type SwapConfig = {
  maxSlippage: number; // Maximum acceptable slippage for a swap. (default: 10) This is as a percent, not basis points;
};
```


https://github.com/coinbase/onchainkit/blob/main/src/swap/types.ts#L189-191